lib/java_buildpack/framework/app_dynamics_agent.rb

```diff
@@ -83,7 +84,7 @@ def host_name(java_opts, credentials)
 
       def node_name(java_opts, credentials)
         name = credentials['node-name'] || @configuration['default_node_name']
-        java_opts.add_system_property('appdynamics.agent.nodeName', "#{name}")
+        java_opts.add_system_property('appdynamics.agent.nodeName', @appdynamics_tier_name + ":#{name}")
```

Comment:
@youngm Can you weigh in on this a little.  Do you think this will help visualization in large CF systems?